lib/membrane/rtp/session_bin.ex

```diff
@@ -361,23 +381,27 @@ defmodule Membrane.RTP.SessionBin do
       |> via_out(Pad.ref(:output, ssrc))
       |> to(rtp_stream_name)
 
-    acc = {new_children, router_link}
+    acc = {new_children, router_link, []}
 
-    {new_children, router_link} =
+    {new_children, router_link, children_actions} =
       extensions
-      |> Enum.reduce(acc, fn {extension_name, config}, {new_children, new_link} ->
+      |> Enum.reduce(acc, fn {extension_name, header_extension_id, config},
```

Comment:
I don't think that those extensions always have something to do with the RTP extensions, it is a bad idea assume so.

Reply:
Maybe we could do something like this:
`extensions` -> `rtp_extensions`
`packet_filters` -> `extensions`
And use `packet_filters` as general extensions instead?

Reply:
Yup, sounds great

Reply:
Renamed in 512ca6b